src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPageCount = Math.ceil(total / perPage);
+  const pages: number[] = [];
+
+  for (let i = 1; i <= totalPageCount; i += 1) {
+    pages.push(i);
+  }
```

Comment:
I think here is better to use function from  `utils` folder to prepare pages.

Reply:
agree